mu/harness/pm.py

```diff
@@ -172,6 +176,23 @@ def sim_cat(self):
                 catalog[sim_name] = sim_class
         return catalog
 
+    def new_io(self, simio):
+        if simio.name in self.simios:
+            raise ValueError('SimIo name taken')
+        self.simios[simio.name] = simio
+
+    def get_io(self, name):
+        return self.simios[name].get()
+
+    def get_all_io(self):
+        ret = {}
+        for name, simio in self.simios:
```

Comment:
If I do:
```
musrv -bg
muctl start PedalBoard
muctl get
```
then I get `too many values to unpack (expected 2)` and I suspect it might be because you need to iterate over `self.simios.items()` rather than just `self.simios` here.